### DISCUSSION TITLE
where is the file /etc/containers/containers.conf in podman 4.4.4？

### DISCUSSION THREAD

**Discussion by peace4j** [containers/podman#18218] [General]
I installed a low version of podman from yum repository many years ago and upgraded it to podman 3.4.4 years ago. Month ago I upgarded it to podman 4.4.4 with yum again. Everything looks fine and runs well until some day I found in "podman info" that my networkBackend=cni then I wanted to modify it in the containers.conf. But wait, I can't find it at /etc/containers/. After searched with "find / -name containers.conf", I realized that nowhere to find such a file in my system.
Why ? 

Any tips would be appreciated!

JJ

**Comment by vrothberg** [comment]
Thanks for reaching out, @peace4j.

Can you share which distribution you are using?  Ultimately, it's up the distribution to decide which configs (not) to ship.  Your distribution of choice may ship a `/usr/share/containers/containers.conf`?

**Reply by peace4j** [reply]
Hello @vrothberg ,

It's CentOS Linux release 7.9.2009 (Core).  
And I checked at /usare/share/containers/, no containers.conf.

find / -name containers.conf 
got file lists:
/usr/local/podman-3.4.4/test/e2e/config/containers.conf
/usr/local/podman-3.4.4/vendor/github.com/containers/common/pkg/config/containers.conf
/usr/local/podman-3.4.4/contrib/remote/containers.conf
/usr/local/podman-3.4.4/contrib/podmanimage/stable/containers.conf

Now it's podman 4.4.4. I upgraded it from 3.4.4 with YUM directly:  yum upgrade podman.
I tried every containers.conf in the list, to modify it with some new configuration items, and found nothing changed to the system. Of course I reboot the host after I did so.

**Reply by vrothberg** [reply]
I assume you used 3rd part repository to install Podman 4.4.4 on CentOS 7.

`/etc/containers/containers.conf` should always be read by Podman, so changes should take effect.

**Reply by peace4j** [reply]
I installed it from epel repository.
OK, I know now /etc/containers/containers.conf is the right file. I will try it later.

**Comment by Luap99** [comment]
Podman doesn't need this config file, it will just work without it. If you want to change something just create the file and set the option you want. The file can be found here: https://github.com/containers/common/blob/main/pkg/config/containers.conf, by default everything is commented out.

**Reply by peace4j** [reply]
Thanks for the info. I will fetch an example from there to learn more.
